packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -1436,6 +1436,20 @@ addFieldsDict(Users, {
     canRead: ['guests'],
     ...schemaDefaultValue(0)
   },
+
+  tagRevisionCount: {
+    ...denormalizedCountOfReferences({
+      fieldName: "tagRevisionCount",
+      collectionName: "Users",
+      foreignCollectionName: "Revisions",
+      foreignTypeName: "revision",
+      foreignFieldName: "userId",
+      filterFn: revision => revision.collectionName === "Tags"
+    }),
+    canRead: ['guests'],
+    ...schemaDefaultValue(0)
```

Comment:
The `denormalizedCountOfReferences` function already comes with the `schemaDefaultValue`, so you can ommit this